_includes/docs/latest/required_roles.md

```diff
@@ -1,27 +1,30 @@
-**Granted at the Organization level**
-
- `forseti_inventory` and `forseti_scanner` require:
+**Forseti server service account requires:
  
+***Granted at the Organization level**
+  
+ * `roles/appengine.appViewer`
  * `roles/bigquery.dataViewer`
  * `roles/browser`
+ * `roles/cloudsql.viewer`
  * `roles/compute.networkViewer`
+ * `roles/compute.securityAdmin`
  * `roles/iam.securityReviewer`
- * `roles/appengine.appViewer`
  * `roles/servicemanagement.quotaViewer`
- * `roles/cloudsql.viewer`
+ * `roles/serviceusage.serviceUsageConsumer`
  
- `forseti_enforcer` requires:
+***Granted at the project level**
  
- * `roles/compute.securityAdmin`
-
-**Granted on the project where Forseti Security is deployed**
-
- `forseti_inventory`, `forseti_scanner`, and `forseti_enforcer` require:
-
+ * `roles/cloudsql.client`
+ * `roles/logging.logWriter`
  * `roles/storage.objectViewer`
  * `roles/storage.objectCreator`
- * `roles/logging.logWriter`
+
+
+***Forseti server service account requires:
```

Comment:
- These changes are confusing as you have three sections for "Forseti server service account requires".
- You also have too many `*`'s in some cases
- This change also seems in conflict with what I did in this [PR](https://github.com/GoogleCloudPlatform/forseti-security/pull/1610)